index.html

```diff
@@ -390,6 +398,13 @@ <h3 class="grouping-title">Events</h3>
               <div>load</div>
               <p>Fired when a model is loaded. Can fire multiple times per <span class="attribute">&lt;model-viewer&gt;</span> if the src attribute is changed.</p>
             </li>
+            <li>
+              <div>model-visibility</div>
+              <p>This event is fired when the visibility of the model changes. When the
+              model is loaded, rendered and fully visible (e.g., the poster is no longer
+              visible), this event will fire and <span class="attribute">event.detail.visible</span>
+              will be "true".</p>
```

Comment:
This event will still fire if the model is not in the viewport, right? We might want to mention that.